stackdriver/queue_manager_test.go

```diff
@@ -257,7 +258,7 @@ func TestSampleDeliveryTimeout(t *testing.T) {
 	c := NewTestStorageClient(t)
 	cfg := config.DefaultQueueConfig
 	cfg.MaxShards = 1
-	cfg.BatchSendDeadline = 100 * time.Millisecond
+	cfg.BatchSendDeadline = model.Duration(100 * time.Millisecond)
```

Comment:
why do we need to update the `time.Duration` to `model.Duration` here?

Reply:
Because Prometheus changed the types used for duration, and cfg is of type `DefaultQueueConfig`, which is provided by Prometheus.